src/main/java/net/minecraftforge/fmp/capabilities/ISlottedCapabilityProvider.java

```diff
@@ -0,0 +1,14 @@
+package net.minecraftforge.fmp.capabilities;
+
+import net.minecraft.util.EnumFacing;
+import net.minecraftforge.common.capabilities.Capability;
+import net.minecraftforge.fmp.multipart.PartSlot;
+
+public interface ISlottedCapabilityProvider
+{
+    
+    public boolean hasCapability(Capability<?> capability, PartSlot slot, EnumFacing facing);
+
+    public <T> T getCapability(Capability<T> capability, PartSlot slot, EnumFacing facing);
```

Comment:
Are modders suposed to look here? Also, new params go to the end of the function. Where would we get the slot, why would this be needed? How would we expand this to entities/items/whatever.